crates/types/src/logs/mod.rs

```diff
@@ -55,6 +55,10 @@ impl LogId {
     pub const fn new(value: u32) -> Self {
         Self(value)
     }
+
+    pub fn as_u32(self) -> u32 {
```

Comment:
I think this is not needed since LogId already drives `derive_more::Into`

Reply:
Ooh, totally missed that! Thank you for pointing that out, I must learn to watch out for those.